src/goto-symex/goto_symex.h

```diff
@@ -140,6 +130,33 @@ class goto_symext
   bool should_pause_symex;
 
 protected:
+
+  struct infot {
```

Comment:
I'm not sure `infot`/`info` is a great name, I'd have chosen `configurationt`/`configuration`.

Reply:
One problem: easily confused with the global `config`. How about `symex_config`?